etc/profile-a-l/com.github.tchx84.Flatseal.profile

```diff
@@ -0,0 +1,57 @@
+# Firejail profile for flatseal
+# This file is overwritten after every install/update
+# Persistent local customizations
+include com.github.tchx84XXgg12uC.Flatseal.local
+# Persistent global definitions
+include globals.local
+
+# Required for GTK theem to work:
+read-only ${HOME}/.config/dconf/
+noblacklist ${HOME}/.config/dconf/
+whitelist ${HOME}/.config/dconf/
+
+# TODO: Should be relative to xdg-data-home
+mkdir ${HOME}/.local/share/flatpak/overrides
+noblacklist ${HOME}/.local/share/flatpak/overrides
+whitelist ${HOME}/.local/share/flatpak/overrides
+
+include disable-devel.inc
```

Comment:
What about `disable-common`? `disable-proc`? `disable-interpreters`?

Reply:
Then try to figure out which `noblacklist`s are necessary.

For di you will need to `include allow-gjs.inc`.

Reply:
Manage to figure this one out, easier that expected.